chart/templates/deploy-imgctrl.yaml

```diff
@@ -21,8 +21,6 @@ spec:
       - name: imgctrl
         image: {{ .Values.image }}
         imagePullPolicy: Always
-        command:
-          - /usr/local/bin/imgctrl
```

Comment:
This is what is causing the e2e to stop working. You gonna have to adjust [here](https://github.com/shipwright-io/image/blob/main/.github/workflows/pullrequest.yaml#L79-L107) to also use ko

Reply:
Sure! Just added a `ENTRYPOINT` on the `Containerfile`, please consider.